1. Desktop app calls Web API/AppCreationScripts/Cleanup.ps1

```diff
@@ -44,27 +44,28 @@ This function removes the Azure AD applications for the sample. These applicatio
         $tenantId = $creds.Tenant.Id
     }
     $tenant = Get-AzureADTenantDetail
-    $tenantName =  ($tenant.VerifiedDomains | Where { $_._Default -eq $True }).Name
+    $tenantName =  ($tenant.VerifiedDomains | Where-Object { $_._Default -eq $True }).Name
     
     # Removes the applications
     Write-Host "Cleaning-up applications from tenant '$tenantName'"
 
     Write-Host "Removing 'service' (TodoListService (active-directory-dotnet-native-aspnetcore-v2)) if needed"
-    $app=Get-AzureADApplication -Filter "DisplayName eq 'TodoListService (active-directory-dotnet-native-aspnetcore-v2)'"  
+    $apps=Get-AzureADApplication -Filter "DisplayName eq 'TodoListService (active-directory-dotnet-native-aspnetcore-v2)'"  
 
-    if ($app)
+    foreach ($app in $apps) 
```

Comment:
Tested that we don't need the if when using the foreach...

Reply:
so why did you keep it? for visibility, @bgavrilMS ?

Reply:
I don't understand your question, I have deleted the `if ($app)` line and repalced with the `foreach`

Reply:
oh, I understand. We don't need the `if` if we have the `foreach`